### PR TITLE
fix: use strategies for cargo workspace plugin (rust)

### LINKED ISSUE #2318: `cargo-workspace` - package missing `version` header in `changelog` when only dependency is updated
When package has only dependency update the changelog looks like this (both in CHANGELOG.md and pr body)
(note `ck3oop-ui-rs: 0.0.1` missing version header`)

:robot: I have created a release *beep* *boop*
---

<details><summary>ck3oop-ui: 0.1.0</summary>

### Features

* **rust-core:** Mod file parsing ([03ce8e1](https://github.com/bukowa/ck3oop/commit/03ce8e167ee99b0c594b4fb3325e7c62963b0825))
</details>

<details><summary>ck3oop-core-rs: 0.1.0</summary>

### Features

* **rust-core:** Mod file parsing ([03ce8e1](https://github.com/bukowa/ck3oop/commit/03ce8e167ee99b0c594b4fb3325e7c62963b0825))
</details>

<details><summary>ck3oop-ui-rs: 0.0.1</summary>

### Dependencies

* The following workspace dependencies were updated
  * dependencies
    * ck3oop-core-rs bumped from 0.0.0 to 0.1.0
</details>

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### PR DESCRIPTION
**Pull request by bukowa**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/release-please/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #2318 🦕

Basically copied code from https://github.com/googleapis/release-please/pull/2112

### CODE REVIEW

**Comment by chingor13** [comment]
Closing and reopening to trigger CI

**Review by chingor13 — APPROVED** [review]
